README.md

```diff
@@ -95,40 +76,71 @@ black ./
+                        Which Dune Client version to use (legacy or official)
+  --consolidate-transfers CONSOLIDATE_TRANSFERS
+                        Flag to indicate whether payout transfer file should be optimized (i.e. squash transfers having same receiver-token pair)
+  --dry-run DRY_RUN     Flag indicating whether script should not post alerts or transactions. Only relevant in combination with --post-tx TruePrimarily intended for
+                        deployment in staging environment.
 ```
-and for the next one:
+
+The solver reimbursements are executed each Tuesday with the accounting period of the last 7 days.
+The default accounting period is 7 days with end date equal to the current date.
+If the payout script can not be run on Tuesday, one will have to specify the start date to specify the correct
+accounting period.
+
+To generate the CSV Transfer file manually run the "quickstart" variant of the script.
+A more fine-tuned variant of the script execution could look like this:
+
 ```shell
-python -m src.fetch.transfer_file --start '2022-03-08'
+python -m src.fetch.transfer_file --start 2023-03-14 --post-cip20 True --post-tx True
 ```
 
+which would run for the accounting period March 14 - 21, 2023, using the Post CIP-20 reward scheme and post the payout
+transaction directly to the safe (i.e. without generating a CSV file).
+
+## Validating the Payout Transaction
 
-Please double-check that the payout is reasonable. That means the eth sent should be between 30-80 ETH, depending on gas prices from last week. Also the amount of cow send should reflect 100x the amount of batches. Reasonable COW totals are around 300000-500000, that means 500-700 batches a day.
+Please visit this [Notion document](https://www.notion.so/cownation/Solver-Payouts-3dfee64eb3d449ed8157a652cc817a8c).
+In particular
+see [Validation by Example](https://www.notion.so/cownation/Solver-Payouts-3dfee64eb3d449ed8157a652cc817a8c?pvs=4#5a99004c03714f939cd80ef41a3d9590)
+section.
 
-Also, it might happen that the slippage of a solver is bigger than the ETH payout. In this case, please do not proceed with the payout, until the root cause is known. Feel free to reach out the project maintainers to do the investigation.
+### Additional Notes
 
-Note that we must wait some time after the period has ended for some data to finalize (e.g. `prices.usd`, `ethereum.transactions` our event data, etc...). Hence, the scripts should not be executed immediately after the accounting period has ended.
+Also, it might happen that the slippage of a solver is bigger than the ETH payout. In this case, please do not proceed
+with the payout, until the root cause is known. Feel free to reach out the project maintainers to do the investigation.
 
-After generating the transfer file and double-checking the results, please create the multi-send transaction with the link provided in the console.
+Note that we must wait some time after the period has ended for some data to finalize (
```

Comment:
Can't say for sure exactly since we do not control their services. Generally,they wait for block finalization before adding event data (about 12 to 15 minutes or 65 blocks). Also, since our queries involve external price data, there may be a lag of about an hour on that. Finally, since our trades table is incrementally populated from sources invoking both event and price data, there may also be a delay. Generally this is why we don't run the scripts until 9am the next day. That is 9 hours after we're expecting data to be available. 

If we wanted to be extra safe, we could probably run a query on latest populated block number for all the relevant sources.